Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -268,8 +271,12 @@ public final class SinglePromptSecureEnclaveValet: NSObject {
     /// the keychain by the operating system due to a face, fingerprint, or password change.
     private func continuedAuthenticationKeychainQuery() throws -> [String : AnyHashable] {
         var keychainQuery = baseKeychainQuery
-        keychainQuery[kSecUseAuthenticationContext as String] = localAuthenticationContext
-        return keychainQuery
+        #if os(tvOS)
+            // tvOS doesn't support LAContext
```

Comment:
I'm assuming omitting the authentication context won't change any behavior. Presumably this never worked on tvOS, but I don't know if the presence of the key affects behavior. cc @dfed

Reply:
So I think we need to make the entire `SinglePromptSecureEnclaveValet` unavailable on tvOS.

The approach taken here will make this type behave like the `SecureEnclaveValet`, which isn't what we want.

The more I think about this, the more I think we make this a breaking change and roll a major version. We may also need a way to let folk on tvOS using this type migrate to `SecureEnclaveValet`, but given that no one has told us this wasn't working I kinda doubt anyone tried to use it that way... so maybe we don't build that until someone asks for it.

Taking the route of making `SinglePromptSecureEnclaveValet` unavailable on tvOS will also force us to solve #270, which is kinda a nice bonus.

Reply:
I agree, I'll update the PR to make `SinglePromptSecureEnclaveValet` unavailable on tvOS.